os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -203,7 +205,12 @@ static void rtl8730e_lcd_put_area(u8 *lcd_img_buffer, u32 x_start, u32 y_start,
 #endif
 	LCDC_LayerConfig(pLCDC, LCD_LAYER, &lcdc_init_struct.layerx[LCD_LAYER]);
 	DCache_CleanInvalidate((u32)lcd_img_buffer, LCDC_IMG_BUF_SIZE);
-	sem_wait(&g_next_frame_block);
+
+	do {
+		ret = sem_wait(&g_next_frame_block);
+		DEBUGASSERT(ret == OK || errno == EINTR);
+	} while (ret < 0);
```

Comment:
The ret is twice checking. how about below?
```suggestion
	while (sem_wait(&g_next_frame_block) != OK) {
		DEBUGASSERT(errno == EINTR);
	}
```